sample-app/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+// This is enabled to make `@Parcelize` tag to work properly.
```

Comment:
Nit: I think that's clear from the code (the string says "parcelize"), no need to comment.